sympy/core/tests/test_assumptions.py

```diff
@@ -726,7 +726,6 @@ def test_Mul_is_infinite():
     assert (x*i).is_finite is None
     assert (f*i).is_finite is False
     assert (x*f*i).is_finite is None
-    assert (z*i).is_finite is False
```

Comment:
I personally think it fits the surrounding tests: if you remove it now and I read those tests later, I would wonder about `(z*i).is_finite`.